ruby_connectors/spec/github_connector/client_spec.rb

```
@@ -73,4 +64,20 @@
       end.to raise_error(ArgumentError, "Unsupported stream: unsupported_stream")
     end
   end
+
+  describe "#catalog" do
+    it "returns a hash of schemas" do
+      result = subject.catalog
```

rubocop

[convention] Name your test subject if you need to reference it explicitly.

(RSpec/NamedSubject)